main.cpp

```diff
@@ -22,16 +22,10 @@ int main (int argc, char** argv) {
   Monster monster1 = Monster::readJson(argv[1]);
   Monster monster2 = Monster::readJson(argv[2]);
 
-  while ( (monster1.getHp() > 0) && (monster2.getHp() > 0) ) {
-
-    if (monster1.getHp() > 0) {
-      monster1.monsterAttack(monster2);
-    }
-
-    if (monster2.getHp() > 0) {
-      monster2.monsterAttack(monster1);
-    }
-
+  while ( (monster1.getHp() > 0) && (monster2.getHp() > 0) )
+  {
+    monster1.monsterCd(monster2);
+    monster2.monsterCd(monster1);
```

Comment:
Mi történik akkor ha monster2 elhalálozik a monster1 támadása után?

Reply:
@vizvezetek Én azt, hogy tudom elérni? Nem csak a repo adminja tudja?

Reply:
@hmartin28 Create an Issue. Aztán be lesz állítva.

Reply:
@vizvezetek beraktam a feltételt mainbe.